tedana/model/fit.py

```diff
@@ -208,6 +217,19 @@ def fitmodels_direct(catd, mmix, mask, t2s, t2s_full, tes, combmode, ref_img,
     else:
         mmix_new = mmix
 
+    if verbose:
```

Comment:
It looks like some of these `filewrite` calls use `mask`, while others use `t2s != 0`. Is there a reason to use both, here ?

Reply:
Mask is overwritten with `t2s != 0` when we moved from selcomps v3 to v2, but I have raised an issue about it in #175. I can do them all here with `mask`, though, for consistency.